scripts/avg_checkpoints.py

```diff
@@ -15,12 +15,15 @@
 import numpy as np
 import tensorflow as tf
 
-from neuralmonkey.logging import log
-
+from neuralmonkey.logging import log as _log
 
 IGNORED_PATTERNS = ["global_step"]
 
 
+def log(message: str, color: str = "blue") -> None:
+    _log(message, color)
+
```

Comment:
Co je tohle?

Reply:
To jsem tam udělal stejně jako je to v import nematus, aby to logovalo modře, abysme měli nějakou štábní kulturu.

Reply:
Jo, prve jsem si nevšiml že to je ve `scripts/`...